build/proj/Build.proj

```diff
@@ -1,74 +1,49 @@
 <!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information. -->
 <Project DefaultTargets="ClearNuGetCache;PrepareMachine;CoreBuild;UploadCodeCoverage">
-  
+
   <Import Project="..\import\Versions.props"/>
   <Import Project="..\import\NuGet.props"/>
   <Import Project="..\import\RepoLayout.props" />
-  
+
   <PropertyGroup>
     <ToolsetBuildProjectFilePath>$(NuGetPackageRoot)RoslynTools.RepoToolset\$(RoslynToolsRepoToolsetVersion)\tools\Build.proj</ToolsetBuildProjectFilePath>
-    <IsCodeCoverageBuild Condition="'$(CODECOV_TOKEN)' != '' AND '$(Configuration)' == 'Debug'">true</IsCodeCoverageBuild>    
+    <IsCodeCoverageBuild Condition="'$(CODECOV_TOKEN)' != '' AND '$(Configuration)' == 'Debug'">true</IsCodeCoverageBuild>
   </PropertyGroup>
 
-  <Target
-    Name="ClearNuGetCache"
-    Condition="'$(ClearNuGetCache)' == 'true'">
-
-    <Message
-      Text="Clearing '$(NuGetPackageRoot)'..."
-      Importance="high" />
+  <Target Name="ClearNuGetCache" Condition="'$(ClearNuGetCache)' == 'true'">
+    <Message Text="Clearing '$(NuGetPackageRoot)'..." Importance="high" />
```

Comment:
In these .proj files, there's a lot of horizontal flattening I did as carriage returns for every element made the file tall and skinny, which was more difficult for me to read. So, you'll see this collapsing throughout.